recipes/cellsnake/post-link.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cellsnake --install-packages
+Rscript -e 'require(randomcoloR)'
```

Comment:
All dependent packages must be installed via conda, so please remove this. If a package is not yet available on conda, they'll need to be added. Biomedical research related packages go to bioconda; general-purpose (like randomcoloR) go to conda-forge.